includes/class-widget.php

```diff
@@ -66,10 +66,24 @@ public function widget( $args, $instance ) {
 		}
 
 		echo '<div class="license">';
-			echo sprintf(
-				'<p><button ' . ( $is_amp ? 'on="tap:republication-tracker-tool-modal"' : '' ) . ' name="%1$s" id="cc-btn" class="republication-tracker-tool-button">%1$s</button></p>',
-				esc_html__( 'Republish This Story', 'republication-tracker-tool' )
-			);
+
+			if ( ! empty( $instance['layout'] ) ) {
```

Comment:
For backwards compatibility, let's use `modal` as the default layout if no layout has been saved. Otherwise nobody gets a button rendered until they go back in and re-save their widget